Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -2363,6 +2363,7 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
     struct DeferredDidReceiveMouseEvent {
```

Comment:
Nit - maybe we can rename this to something along the lines of `DeferredMouseEventCompletionHandler`?